content/docs/latest/spire-helm-charts-hardened-about/identifiers.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Identifiers
+short: Identifiers
+description: How to configure SPIFFE IDs
+kind: spire-helm-charts-hardened-about
+weight: 200
+aliases:
+    - /docs/latest/helm-charts-hardened/identifiers
+---
+
+## Defaults
+
+By default the chart deploys the SPIRE Controller Manager that enables management of SPIFFE Identifiers by Kubernetes Custom Resources.
+
+The chart itself by default deploys a ClusterSPIFFEID Custom Resource that gives an identifier to all pods.
+
+Out of the box, in a lot of use cases you do not need to add additional identifiers.
```

Comment:
```suggestion
- The chart deploys [SPIRE Controller Manager](https://github.com/spiffe/spire-controller-manager) to manage SPIFFE Identifiers by Kubernetes Custom Resources.
- The chart deploys a ClusterSPIFFEID Custom Resource that gives an identifier to all pods or the form `spiffe://{{ .TrustDomain }}/ns/{{ .PodMeta.Namespace }}/sa/{{ .PodSpec.ServiceAccountName }}`.
- For a lot of use cases you don't need to add additional identifiers.
```

Reply:
Should we remove the 'or' in line 14 or was it intended to be 'of'?

Reply:
Should be of